### PR TITLE
VUFIND-1667: Fix tabbing within a modal dialog

### PR DESCRIPTION
**Pull request by maccabeelevine**
For https://openlibraryfoundation.atlassian.net/browse/VUFIND-1667

Disable the "Modal Extension" event listener in bootstrap-accessibility(.min).js.  Lightbox.js does the same idea better in `retainFocus`.

### CODE REVIEW

File: themes/bootstrap3/js/lightbox.js
Comment (maccabeelevine) [inline themes/bootstrap3/js/lightbox.js]: The ugly variable names are just echoing what's in bootstrap-accessibility, for easier reference.

File: themes/bootstrap3/js/lightbox.js
Comment (demiankatz) [inline themes/bootstrap3/js/lightbox.js]: Looks like eslint wants spaces around the `-` in this line; not sure why the auto-generated message is getting attached to the wrong line of code when it displays here.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Looks like one small thing is needed to make continuous integration happy.

Beyond that, I defer to @crhallberg and @EreMaijala for overall approval of this. (Makes sense to me, but want to be sure we do it in a way that can be most easily removed as part of the bootstrap5 migration).

File: themes/bootstrap3/js/lightbox.js
Comment (maccabeelevine) [reply themes/bootstrap3/js/lightbox.js]: Fixed

**Comment by demiankatz** [comment]
Thanks, @maccabeelevine! I'll now look forward to hearing from @crhallberg and/or @EreMaijala when their time permits.

File: themes/bootstrap3/js/lightbox.js
Comment (crhallberg) [inline themes/bootstrap3/js/lightbox.js]: Well sussed! I notice that it applied to the first element as well. Might as well take care of both, right?

https://github.com/paypal/bootstrap-accessibility-plugin/blob/main/src/js/modal.js#L16

Comment (maccabeelevine) [reply themes/bootstrap3/js/lightbox.js]: Sure, I made that change, I don't think there is any impact but I agree it's better to remove both listeners since we really want to fully replace the original function.

**Review by demiankatz — APPROVED** [review]
Looks fine to me. I'll wait for confirmation from @crhallberg before the final merge.

**Review by EreMaijala — APPROVED** [review]

**Pull request merged**
